### ISSUE TITLE
Connect esp32 cam

### ISSUE THREAD

**Issue by skshadan** [NelsenEW/espdrone-nh#2]
hey, is this repo still working? also can i connect an esp-32cam??

**Comment by NelsenEW** [comment]
Yes the repo is working with the version we have tested, we havent tested with the newer version of espidf

**Comment by skshadan** [comment]
Hey Nelson, thanks for replying. I need some help from you regarding the ESP32-CAM. I've also emailed you about this. It would be great if we could connect to discuss it further. 

Thanks for the amazing work!

**Comment by skshadan** [comment]
hey, i have esp32 wroom, now i want to connect ov2640  camera module, how can i connect with it?

**Comment by skshadan** [comment]
[1197/1199] Linking CXX executable ESPDrone.elf
FAILED: ESPDrone.elf
cmd.exe /C "cd . && D:\nh\idf-nh\tools\xtensa-esp32-elf\esp-2020r3-8.4.0\xtensa-esp32-elf\bin\xtensa-esp32-elf-g++.exe  -mlongcalls -Wno-frame-address   @CMakeFiles\ESPDrone.elf.rsp  -o ESPDrone.elf  && cd ."
d:/nh/idf-nh/tools/xtensa-esp32-elf/esp-2020r3-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld.exe: ESPDrone.elf section `.rtc.bss' will not fit in region `rtc_slow_seg'
d:/nh/idf-nh/tools/xtensa-esp32-elf/esp-2020r3-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld.exe:esp32_out.ld:109: undefined symbol `_bss_end' referenced in expression
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
ninja failed with exit code 1
PS D:\nh\espdrone-nh>


hey, i am getting this issue, tell me how to solve this issue?
i have changed the esp32.project.Id.in also.
